app/wear/src/main/kotlin/io/github/xxfast/nytimes/wear/screens/home/HomeScreen.kt

```diff
@@ -1,38 +1,50 @@
 package io.github.xxfast.nytimes.wear.screens.home
 
 import androidx.compose.runtime.Composable
-import com.arkivanov.decompose.extensions.compose.jetbrains.stack.animation.slide
-import com.arkivanov.decompose.extensions.compose.jetbrains.stack.animation.stackAnimation
-import com.arkivanov.decompose.router.stack.pop
+import androidx.compose.ui.graphics.Color
 import com.arkivanov.decompose.router.stack.push
-import io.github.xxfast.krouter.RoutedContent
 import io.github.xxfast.krouter.Router
 import io.github.xxfast.krouter.rememberRouter
+import io.github.xxfast.nytimes.models.ArticleUri
+import io.github.xxfast.nytimes.models.TopStorySection
 import io.github.xxfast.nytimes.screens.home.StoryHomeScreen
 import io.github.xxfast.nytimes.screens.home.StoryHomeScreen.Details
 import io.github.xxfast.nytimes.screens.home.StoryHomeScreen.List
+import io.github.xxfast.nytimes.wear.screens.navigation.SwipeDismissContent
 import io.github.xxfast.nytimes.wear.screens.story.StoryScreen
 import io.github.xxfast.nytimes.wear.screens.topStories.TopStoriesScreen
 
 @Composable
 fun HomeScreen() {
   val router: Router<StoryHomeScreen> = rememberRouter(StoryHomeScreen::class, listOf(List))
 
-  RoutedContent(
-    router = router,
-    animation = stackAnimation(slide())
-  ) { screen ->
-    when (screen) {
-      List -> TopStoriesScreen(
-        onSelectArticle = { section, uri, title -> router.push(Details(section, uri, title)) }
-      )
+  SwipeDismissContent(router = router) {
+    ContentScreen(screen = it,
+      onSelectArticle = { section, uri, title -> router.push(Details(section, uri, title)) })
+  }
+}
+
+@Composable
+internal fun Color.fudge(fudgeFactor: Int): Color {
+  // TODO fix in wear compose
+  // without this the wrong state is remembered in SwipeToDismissBox
+  return this.copy(alpha = 1f - ((fudgeFactor % 2) * 0.01f))
+}
```

Comment:
Looks like this is only used in `SwipeDismissContent` once - so I've written this inline 
```kotlin
// TODO fix in wear compose
// without this the wrong state is remembered in SwipeToDismissBox
val fudgeFactor: Int = stack.items.size
val fudgeAmount: Float = 1f - ((fudgeFactor % 2) * 0.01f)
val backgroundScrimColor: Color = MaterialTheme.colors.background.copy(alpha = fudgeAmount)

SwipeToDismissBox(
  ..
  backgroundScrimColor = backgroundScrimColor,
)
```